uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,31 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Gets a map of contact roles and IDs
+   *
+   * @return {Promise} resolved with {Object}
+   */
+  function getRolesAndIDs () {
+    return new Promise(function (resolve) {
+      var rolesAndIDs = {};
+
+      exec('cv api contact.get', {maxBuffer: 1024 * 1000}, function (err, result) {
+        if (err) {
+          console.log(color('Unable to fetch contact roles and IDs: ' + err, 'RED'));
+        }
+
+        _.each(JSON.parse(result).values, function (contact) {
```

Comment:
if you use the `sequential=1` property in the api call, then `.values` becomes an array and you can simply do
```js
JSON.parse(result).values.forEach(function (contact) {

});
```